app_dart/lib/src/request_handlers/check_flaky_tests_and_update_github_utils.dart

```diff
@@ -0,0 +1,122 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:core';
+
+import 'package:github/github.dart';
+
+import '../service/bigquery.dart';
+
+const String _commitPrefix = 'https://github.com/flutter/flutter/commit/';
+const String _buildPrefix = 'https://ci.chromium.org/ui/p/flutter/builders/prod/';
+
+// Issue
+const String kCloseIssueComment = '''
+Closed this issue because the flaky rate of this test has dropped to 0.0% and is considered
+to be no longer flaky.
+
+If you think otherwise, reopen this issue and assign an owner to prevent it from being closed.
+''';
+const String kTeamFlakeLabel = 'team: flakes';
+const String kSevereFlakeLabel = 'severe: flake';
+const String kP1Label = 'P1';
+const String kP2Label = 'P2';
+
+class IssueBuilder {
+  IssueBuilder({
+    this.statistic,
+    this.threshold,
+    this.isImportant,
+  });
+
+  final BuilderStatistic statistic;
+  final double threshold;
+  final bool isImportant;
+  static final RegExp issueTitleRegex = RegExp(r'(?<name>.+) is (?<rate>.+)% flaky');
+
+  String get issueTitle {
+    return '${statistic.name} is ${_formatRate(statistic.flakyRate)}% flaky';
+  }
+
+  String get issueBody {
+    return '''
+$_issueSummary
```

Comment:
Do we need to distinguish the two cases: exceeding slo and the top one below slo?

Reply:
yes, the issueSummary getter does that.